src/LibraryViewExtensionMSWebBrowser/LibraryViewExtensionMSWebBrowser.csproj

```diff
@@ -24,7 +24,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
+    <OutputPath>..\..\bin\AnyCPU\Debug\</OutputPath>
```

Comment:
I don't think this should usually be set directly like this, instead import our normal configuration like in other csproj files and delete these lines, then the output path will be set correctly for each config without needing to update it.

```
  <Import Project="$(SolutionDir)/Config/CS.props" />
```

Reply:
for example, your release path is still wrong in this commit.

Reply:
Thanks, will correct this